service_mesh/v2x/ossm-security.adoc

```diff
@@ -18,7 +18,7 @@ include::modules/ossm-security-mtls.adoc[leveloffset=+1]
 
 include::modules/ossm-config-sec-mtls-mesh.adoc[leveloffset=+2]
 
-include::modules/ossm-config-sidecar-mtls.adoc[leveloffset=+3]
+include::modules/ossm-config-sidecar-mtls.adoc[leveloffset=+2]
```

Comment:
In the live build, the first heading in the target module is H3 in the assembly and it is subsequently not rendered in the TOC. Are you intending to increase it by one level? In the preview, the "Configuring sidecars for incoming connections for specific services" heading is parallel to the "Enabling strict mTLS across the service mesh" heading, whereas in the live build it is nested within it.